recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+MACOSX_SDK_VERSION:       # [osx and x86_64]
+  - 10.14                 # [osx and x86_64]
```

Comment:
Is this a requirement for Mono or it's just a standard version we use across all our packages?

Reply:
The version we use on Prefect is `10.10`, while this package requires a minimum of `10.11`. Since I am not sure if `10.11` is available on Prefect I chose to go with this version which we use in a few other packages too.